protocols/migration/migration_helpers.py

```diff
@@ -163,47 +164,60 @@ def migrate_interpretation_request_rd_to_interpreted_genome_latest(json_dict, as
         raise MigrationError("Interpretation Request RD is not in versions: [2.1.0, 3.0.0, 4.0.0, 5.0.0]")
 
     @staticmethod
-    def migrate_interpreted_genome_rd_to_latest(json_dict, assembly, interpretation_request_version):
+    def migrate_interpreted_genome_rd_to_latest(json_dict, assembly=None, interpretation_request_version=None):
         """
         :type json_dict: dict
         :type assembly: Assembly
         :type interpretation_request_version: int
         :rtype: InterpretedGenomeRD_5_0_0
         """
-        ig_v500 = None
+        ig_v600 = None
 
-        if PayloadValidation(klass=InterpretedGenomeRD_5_0_0, payload=json_dict).is_valid:
-            ig_v500 = InterpretedGenomeRD_5_0_0.fromJsonDict(jsonDict=json_dict)
+        if PayloadValidation(klass=InterpretedGenome_6_0_0, payload=json_dict).is_valid:
+            logging.info("Case in models reports 6.0.0")
+            ig_v600 = InterpretedGenome_6_0_0.fromJsonDict(jsonDict=json_dict)
+
+        elif PayloadValidation(klass=InterpretedGenomeRD_5_0_0, payload=json_dict).is_valid:
             logging.info("Case in models reports 5.0.0")
+            ig_v500 = InterpretedGenomeRD_5_0_0.fromJsonDict(jsonDict=json_dict)
+            ig_v600 = MigrateReports500To600().migrate_interpreted_genome_rd(old_instance=ig_v500)
+
+        elif assembly is None or interpretation_request_version is None:
+            raise MigrationError(
+                "Parameters <assembly> and <interpretation_request_version> are required for models earlier than 5.0.0"
+            )
 
-        if PayloadValidation(klass=InterpretedGenomeRD_4_0_0, payload=json_dict).is_valid:
+        elif PayloadValidation(klass=InterpretedGenomeRD_4_0_0, payload=json_dict).is_valid:
+            logging.info("Case in models reports 4.0.0")
             ig_v400 = InterpretedGenomeRD_4_0_0.fromJsonDict(jsonDict=json_dict)
             ig_v500 = MigrateReports400To500().migrate_interpreted_genome_rd(
                 old_instance=ig_v400, assembly=assembly, interpretation_request_version=interpretation_request_version
             )
-            logging.info("Case in models reports 4.0.0")
```

Comment:
Logging is useful in some scenarios. Remember this is not only used by the cipapi

Reply:
It hasn't been removed @priesgo -- it's moved up to line 191

Reply:
did not say anything I see it now up there...